test/createAzureResource.test.ts

```diff
@@ -58,14 +57,74 @@ suite('Create Azure Resources', async function (this: ISuiteCallbackContext): Pr
     });
 
-    test('createFunctionApp (Basic)', async () => {
+    test('Create windows function app (Basic) and deploy JavaScript project', async () => {
+        const functionName: string = 'HttpTrigger';
         resourceGroupsToDelete.push(resourceName1);
-        await runWithFuncSetting('advancedCreation', undefined, async () => {
-            ext.ui = new TestUserInput([resourceName1]);
-            await vscode.commands.executeCommand('azureFunctions.createFunctionApp');
-            const createdApp: WebSiteManagementModels.Site = await webSiteClient.webApps.get(resourceName1, resourceName1);
-            assert.ok(createdApp);
+        await clearProjectFile(projectPath);
+        await runWithFuncSetting('projectLanguage', ProjectLanguage.JavaScript, async () => {
+            await runWithFuncSetting('advancedCreation', undefined, async () => {
+                ext.ui = new TestUserInput([resourceName1]);
+                await vscode.commands.executeCommand('azureFunctions.createFunctionApp');
+                const createdApp: WebSiteManagementModels.Site = await webSiteClient.webApps.get(resourceName1, resourceName1);
+                assert.ok(createdApp);
+                await runWithFuncSetting('projectRuntime', ProjectRuntime.v2, async () => {
+                    const templateId: string = 'HttpTrigger-JavaScript';
+                    const authLevel: string = 'function';
+                    await vscode.commands.executeCommand('azureFunctions.createNewProject', projectPath, ProjectLanguage.JavaScript, ProjectRuntime.v2, false, templateId, functionName, { AuthLevel: authLevel });
+                    await validateProject(projectPath, getJavaScriptValidateOptions(true));
+                });
+            });
+        });
+
+        ext.ui = new TestUserInput([resourceName1, 'Deploy']);
+        await vscode.commands.executeCommand('azureFunctions.deploy');
+        await delay(5000);
+        const getFunction: WebSiteManagementModels.FunctionEnvelopeCollection = await webSiteClient.webApps.listFunctions(resourceName1, resourceName1);
+        assert.equal(getFunction[0].name, `${resourceName1}/${functionName}`);
+        // Initialize the contents of the clipboard as "Undefined" and then get function URL by command
+        clipboard.writeSync(typeof (undefined));
+        ext.ui = new TestUserInput([resourceName1, functionName]);
+        await vscode.commands.executeCommand('azureFunctions.copyFunctionUrl');
+        assert.ok(clipboard.readSync() !== 'undefined', `The value of the copy function url should not be ${clipboard.readSync()}`);
+
+        const result: string = execSync(`curl ${clipboard.readSync()}"&"name=${resourceName1}`).toString();
+        assert.equal(result, `Hello ${resourceName1}`, `The return value should be Hello ${resourceName1}`);
+    });
+
+    test('Deploy CSharp project (windows)', async () => {
+        const resourceName3: string = getRandomHexString().toLowerCase();
```

Comment:
Could you change this to resourceName2 and the one below to resourceName3?  It's a bit confusing that they are out of order.